core/commonMain/src/kotlinx/io/text/TextInputObsolete.kt

```diff
@@ -64,34 +65,34 @@ fun Input.readUTF8LineTo(out: Appendable) {
 /**
  * @throws MalformedInputException
  */
-fun Input.readUTF8String(length: Int): String = buildString(length) {
-    readUTF8StringTo(this, length)
+fun Input.readUtf8String(length: Int): String = buildString(length) {
+    readUtf8StringTo(this, length)
 }
 
 /**
  * @throws MalformedInputException
  */
-fun Input.readUTF8StringUntilDelimiter(delimiter: Char): String = buildString {
-    readUTF8StringUntilDelimiterTo(this, delimiter)
+fun Input.readUtf8StringUntilDelimiter(delimiter: Char): String = buildString {
+    readUtf8StringUntilDelimiterTo(this, delimiter)
 }
 
 /**
  * @throws MalformedInputException
  */
-fun Input.readUTF8StringUntilDelimiters(delimiters: String): String = buildString {
-    readUTF8StringUntilDelimitersTo(this, delimiters)
+fun Input.readUtf8StringUntilDelimiters(delimiters: String): String = buildString {
+    readUtf8StringUntilDelimitersTo(this, delimiters)
 }
 
 /**
  * @throws MalformedInputException
  */
-private inline fun Input.decodeUTF8Chars(consumer: (Char) -> Boolean): Int {
+private inline fun Input.decodeUtf8Chars(consumer: (Char) -> Boolean): Int {
     var byteCount = 0
     var value = 0
-    var state = STATE_UTF8
+    var state = STATE_UTF_8
     var count = 0
 
-    while (state != STATE_FINISH) {
+    while (state != STATE_FINISH && !eof()) {
```

Comment:
Why we expect `eof` here?

Reply:
To allow reading strings that are not terminated with a line break.
Frankly speaking, we should also check that eof didn't happen in the middle of the code point, but I decided not to touch the implementation of text decoding at all for now